src/app/collection/shared/included/curators/curators.component.html

```diff
@@ -0,0 +1,21 @@
+<section class = "section section-curators">
+    <div class="section-accent">
+        <div class="accent-content">
+            <i class="fa-2x fas fa-users"></i>
+            <div class="curators-title">
+             COLLECTION 
+             CURATORS
+            </div>
+            <p class="curators-desc">
+                Collection curators are 
+                responsible for their collection’s 
+                review process and continued quality
+            </p>
+        </div>
+   </div>
+   <div *ngIf="curators && curators.length > 0" class = "curators">
+
+        <clark-curator-card *ngFor="let curator of curators" [curator]="curator"></clark-curator-card>
```

Comment:
I would keep the For a layer before the clark-curator-card and remove the 100px margin-right. The reason why is because if you have 4-5 curators. It is now going to take up close to 500-600px and when a screen is smaller it'll be broken.
I would try
```
<div *ngFor="let curator of curators" class="curator-container">
     <clark-curator-card [curator]="curator"></clark-curator-card>
</div>
```
.curator-container {
     display: flex;
     justify-content: space-evenly;
} //Or something similar to space evenly rather then have a fixed margin in the cards